src/jgikbase/idmapping/cli.py

```diff
@@ -106,11 +101,33 @@ def _list_users(self, local_user_handler: LocalUserHandler, verbose):
             self._stdout.write('{}{}\n'.format(u, ' *' if admin else ''))
         return 0
 
-    def _admin(self, local_user_handler: LocalUserHandler, user: Username, admin: bool, verbose):
+    def _create_user(
+            self,
+            local_user_handler: LocalUserHandler,
+            username: Username,
+            verbose):
+        try:
+            t = local_user_handler.create_user(username)
+        except Exception as e:
+            self._handle_error(e, verbose)
```

Comment:
In this class, rather than catching Exception in a bunch of places, you could make a decorator:

```py
@handle_error
def _create_user(...)
   ...
```

And in the decorator itself you can wrap the function call in a try/except, and you'd also have access to the `verbose` arg